src/scripts/main.js

```diff
@@ -5,4 +5,36 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+people.forEach(person => {
+  const tr = document.createElement('tr');
+  const personName = person.name;
```

Comment:
It's better to use destructuring here and don't create new variables for `name`, `born`, `died`